analytics/analytics/build_timing_processor.py

```diff
@@ -88,6 +90,7 @@ def get_timings_json(job: ProjectJob) -> list[dict]:
 def upload_build_timings(job_input_data_json: str):
     # Read input data and extract params
     job_input_data = json.loads(job_input_data_json)
+    setup_gitlab_job_sentry_tags(job_input_data)
```

Comment:
Should we not just put the invocation to this function in the webhook endpoint itself?

Reply:
It'll be run on a different machine. I briefly tried looking into how to reduce the duplication, I'd probably use a custom celery base task, but it didn't really look easier 😕.

Reply:
Right, I see.